content/en/docs/concepts/policy/limit-range.md

```diff
@@ -1,64 +1,61 @@
 ---
 reviewers:
 - sftim
-title: LimitRange
+- kbhawkey
+title: Limit Ranges
 content_template: templates/concept
 weight: 10
 ---
 
 {{% capture overview %}}
 
-By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on kubernetes cluster.
+By default, containers run with unbounded [compute resources](/docs/user-guide/compute-resources) on a Kubernetes cluster.
 With Resource quotas, cluster administrators can restrict the resource consumption and creation on a namespace basis.
-Within a namespace, a Pod/Container can still consume as much CPU and memory as the defined namespace's resource quota.
-there is a concern that one Pod/Container could monopolise its namespace resources.
+Within a namespace, a Pod or Container can still consume as much CPU and memory as the defined namespace's resource quota.
+there is a concern that one Pod or Container could monopolise its namespace resources.
 LimitRange is a tool to address this concern.
 
 {{% /capture %}}
 
 
 {{% capture body %}}
 
-A limit range, defined by a `LimitRange` object, provides constraints that can :
-- Enforce min/max compute resources usage per Pod/Container in a namespace
-- Enforce min/max storage request per PersistentVolumeClaim in a namespace
+A limit range, defined by a `LimitRange` object, provides constraints that can:
+- Enforce minimun and maximum compute resources usage per Pod or Container in a namespace
+- Enforce minimum and maximum storage request per PersistentVolumeClaim in a namespace
 - Enforce a ratio between request and limit for a resource in a namespace
-- Set default request/limit for compute resources in a namespace and automatically inject them to Containers at runtime 
+- Set default request/limit for compute resources in a namespace and automatically inject them to Containers at runtime.
 
+## Enabling Limit Range 
+
+Limit Range support is enabled by default for many Kubernetes distributions.  It is
+enabled when the apiserver `--enable-admission-plugins=` flag has `LimitRanger` admission controller as
+one of its arguments.
+
+A limit range is enforced in a particular namespace when there is a
+`LimitRange` object in that namespace.
 
 Limit Range work like this:
 
 - The administrator creates one `LimitRange` in one namespace.
-- Users create resources (Pods, Containers, PersistentVolumeClaim) in the namespace,

```

Comment:
nit: Instead of `Limit Range work like this:`, could you add another header. Something like,
## Overview of Limit Range 
nit: lines 42-43 need re-wording

Reply:
Some re-wording